tvm/vm/stack.go

```diff
@@ -331,3 +333,56 @@ func (s *Stack) Copy() *Stack {
 
 	return c
 }
+
+func (s *Stack) CheckUnderflow(values ...int) error {
+	stackLen := len(s.elems)
+	for _, val := range values {
+		if stackLen < val {
+			return vmerr.ErrStackUnderflow
+		}
+	}
+	return nil
+}
+
+func (s *Stack) MustFromTop(offset int) int {
```

Comment:
Must methods not need here, better to use FromTop always and check error, since this is not a part of "public" api, but more like internal tvm thing